source/contribute/documentation.md

```diff
@@ -3,7 +3,7 @@
 This documentation is built with the excellent [GitBook](https://github.com/GitbookIO/gitbook) tool
```

Comment:
This is outdated.

Reply:
We should replace this with a link to the repo maybe?

Reply:
Updated